### PR TITLE
dont grow items in the app list

### LINKED ISSUE #1938: Apps not correctly align on apps page
I really like the latest changes to the apps page, the raster looks much better than the list. But it seems that sometimes the apps are not correctly align, see this screenshot:

apps

I would expect that the "Versions" app is in the second column, under the text editor and not between the second and the third column.

cc @nextcloud/designers

### PR DESCRIPTION
**Pull request by icewind1991**
This keeps the bottom row nicely alligned

Before:

After:

cc @jancborchardt 

fixes #1938

### CODE REVIEW

**Comment by schiessle** [comment]
tested, works great! 👍

**Comment by schiessle** [comment]
@eppfel maybe you want to review it? Thanks!

**Comment by skjnldsv** [comment]
👍

**Comment by skjnldsv** [comment]
👎 
If we have a lower resolution than a multiple of elements per rows, we get empty spaces.
apps nextcloud

**Comment by Espina2** [comment]
I agree @skjnldsv but I prefer to have more space in the right that see the elements misaligned.

**Comment by skjnldsv** [comment]
Or we could implement the code I suggested here?
http://codepen.io/kunji/pen/yNPVVb

**Comment by schiessle** [comment]
> If we have a lower resolution than a multiple of elements per rows, we get empty spaces.

I'm pretty sure that this is only the case because in your example your window misses a few pixel until there is enough space for a third column. I tried it with zooming up/down the browser window and it scaled quite well, adding/removing additional columns depending on the space on the right.

**Comment by skjnldsv** [comment]
@schiessle No no, it switch at 330px only. Flex without grow is like a float. 
kazam_screencast_00000

**Comment by schiessle** [comment]
for me this looks ok, it seems like it switches as soon as there is enough empty space for a additional column.

**Comment by jancborchardt** [comment]
The current implementation looks much better on smaller screens, or iPads or whatever. We should only fix this when we can retain that layout. Otherwise we’re just switching the trade-off.

@skjnldsv so do you want to implement that? :)

**Comment by skjnldsv** [comment]
@jancborchardt on it!

**Comment by eppfel** [comment]
not, yet
I think you can remove the `flex`, if you use percentages for the `width`.
bildschirmfoto 2016-12-02 um 21 11 52

**Comment by skjnldsv** [comment]
@jancborchardt @eppfel Done!
Please review 

I used a small backstep to 2 column when the sidebar gets hidden in mobile view!

kazam_screencast_00001-2

**Comment by skjnldsv** [comment]
@eppfel I had to leave, didn't finished my work ;)

**Comment by MorrisJobke** [comment]
Works nicely 👍

**Comment by eppfel** [comment]
:+1:

File: settings/css/settings.css [hunk 1]
```diff
@@ -231,7 +231,7 @@ table.nostyle td { padding: 0.2em 0; }
 	text-align: right;
 }
 #sessions .token-list td > a.icon,
-#apppasswords .token-list  td > a.icon {
+#apppasswords .token-list	td > a.icon {
```
Comment (eppfel) [inline settings/css/settings.css]: 😂

Comment (skjnldsv) [inline settings/css/settings.css]: Ahahah! :D

**Pull request merged**